commands/FBAccessibilityCommands.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/python
+
+# Copyright (c) 2015, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+import re
+import os
+
+import lldb
+import fblldbbase as fb
+import fblldbobjecthelpers as objHelpers
+
+# This is the key corresponding to accessibility label in _accessibilityElementsInContainer:
+ACCESSIBILITY_LABEL_KEY = 2001
+
+def lldbcommands():
+  return [
+    FBPrintAccessibilityLabels(),
+    FBFindViewByAccessibilityLabelCommand(),
+  ]
+
+class FBPrintAccessibilityLabels(fb.FBCommand):
+  def name(self):
+    return 'pa11y'
+
+  def description(self):
+    return 'Print accessibility labels of all views in hierarchy of <aView>'
+
+  def args(self):
+    return [ fb.FBCommandArgument(arg='aView', type='UIView*', help='The view to print the hierarchy of.', default='(id)[[UIApplication sharedApplication] keyWindow]') ]
+
+  def run(self, arguments, options):
+    forceStartAccessibilityServer();
+    printAccessibilityHierarchy(arguments[0])
+
+class FBFindViewByAccessibilityLabelCommand(fb.FBCommand):
+  def name(self):
+    return 'fa11y'
+
+  def description(self):
+    return 'Find the views whose accessibility labels match labelRegex and puts the address of the first result on the clipboard.'
+
+  def args(self):
+    return [ fb.FBCommandArgument(arg='labelRegex', type='string', help='The accessibility label regex to search the view hierarchy for.') ]
+
+  def run(self, arguments, options):
+    forceStartAccessibilityServer()
+    rootView = fb.evaluateExpression('(id)[[UIApplication sharedApplication] keyWindow]')
+    accessibilityGrepHierarchy(rootView, arguments[0])
+
+def forceStartAccessibilityServer():
+  #We try to start accessibility server only if we don't have needed method active
+  if not fb.evaluateBooleanExpression('[UIView instancesRespondToSelector:@selector(_accessibilityElementsInContainer:)]'):
+    #Starting accessibility server is different for simulator and device
+    if fb.evaluateBooleanExpression('((NSRange)[(NSString *)[(id)[(id)[UIDevice currentDevice] model] lowercaseString] rangeOfString:@"simulator"]).location != __LONG_MAX__'):
+      lldb.debugger.HandleCommand('expr (void)[[UIApplication sharedApplication] accessibilityActivate]')
+    else:
+      lldb.debugger.HandleCommand('expr (void)[[[UIApplication sharedApplication] _accessibilityBundlePrincipalClass] _accessibilityStartServer]')
+
+def accessibilityLabel(view):
+  #using Apple private API to get real value of accessibility string for element.
+  return fb.evaluateExpressionValue('(id)[%s accessibilityAttributeValue:%i]' % (view, ACCESSIBILITY_LABEL_KEY), False)
```

Comment:
Not sure how to do use it with a string. If I try evaluateObjectExpression(...) I get string address, instead of actual string in case or evaluateExpressionValue(...).getObjectDescription(). Any suggestion on how to do it right?